tools/benchmark/.eslintrc.cjs

```diff
@@ -10,6 +10,7 @@ module.exports = {
 	},
 	rules: {
 		"@typescript-eslint/no-shadow": "off",
+		"@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
Can you point out where was this complaining? If we can make the return types explicit, we should, instead of disabling the rule.

Reply:
It's relative to the function map in resultFormatting.ts. The function like (number) => string doesn't have word `return` in the function, which breaks the rule but I think it doesn't make sense to put return in them

Reply:
They don't need to have an explicit `return` but still should have a return _type_ before the `=>` that starts the function body. I.e.
`(parameter: number): string => number.toString()`
instead of just
`(parameter: number) => number.toString()`

Reply:
I see! That's what it means! Updating accordingly. Thx :)